pages/common/keepassxc-cli.md

```diff
@@ -0,0 +1,36 @@
+# keepassxc-cli
+
+> Command Line interface for the keepassxc app.
+> More information: <https://keepassxc.org/>.
+
+- Search entries:
+
+`keepassxc-cli lookup {{database_file}} {{name}}`
```

Comment:
```suggestion
`keepassxc-cli lookup {{path/to/database_file}} {{name}}`
```
This is what we use to indicate, that the file doesn't have to be in the current directory. Please also apply this in the other examles.